packages/terra-clinical-item-collection/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ ChangeLog
 Unreleased
 ----------
 
+3.17.1 - (October 11, 2018)
```

Comment:
The version / date will get updated during the release process, for now this description should be under `Unreleased` with a `Changed` title.

Unreleased
----------
### Changed
* Updated prop descriptions in ItemCollections.jsx and Item.jsx.

Reply:
Oops! Thanks. Updated.